Makefile

```diff
@@ -4,11 +4,39 @@ help:
 	@echo "release       Make new major release of all formulas"
 	@echo "mrconfig      Re-generate .mrconfig with all formulas on Github"
 	@echo "html          Build html documentation"
-	@echo "pdf           Build pdf documentation"
+
+FORMULAS=`for i in 1 2 3 4 5; do curl -s "https://api.github.com/orgs/salt-formulas/repos?page=$$i&per_page=100" |grep -E 'clone_url.*salt-formula-' | sed 's/.*salt-formula-\([-_[:alnum:]]*\).git.*/\1/'; done| xargs -n1 | sort -u `;
```

Comment:
not, its evaluated at the time the variable is used, welcome Makefile
let's keep that for the time we are over 500 formulas